patch/files/scripts/outside.gd

```diff
@@ -608,6 +622,36 @@ func setcharacter(text):
 		if $charactersprite.modulate.a != 1:
 			get_parent().nodeunfade($charactersprite, 0.3)
 	
+func marketsattext():
+	#ralph5
+	var sattext = ""
+	var starttext = ""
+	var endtext = ""
+	var extrazero = ".0"
+	var racetext
+	var marketrate = 1
+	sattext = "\n\nA sign board lists current relative pricing by race:\n"
+	for i in globals.races:
+		marketrate = round(globals.state.racemarketsat[i]*100)/100
+		racetext = i.replace('Beastkin', '')
+		if marketrate < 0.9:
+			starttext = "[color=red]"
+			endtext = "[/color]"
+		elif marketrate >= 2.0:
+			starttext = "[color=green]"
+			endtext = "[/color]"
+		else:
+			starttext = "[color=yellow]"
+			endtext = "[/color]"
+		if !marketrate in [0,1,2,3,4,5,6,7,8,9,10]:
+			sattext = sattext + "Price x " + starttext + str(marketrate) + endtext + " for [color=aqua]" + racetext + endtext + "\n"

```

Comment:
[color=aqua] has no corresponding [/color] in both cases

Reply:
Fixed